scripts/dfx_tests.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```

Comment:
This file should be moved inside the /script/tests folder and should be integrated with the CI